x-pack/plugins/observability/public/plugin.ts

```diff
@@ -114,6 +115,7 @@ export interface ObservabilityPublicPluginsSetup {
   home?: HomePublicPluginSetup;
   usageCollection: UsageCollectionSetup;
   embeddable: EmbeddableSetup;
+  licensing: LicensingPluginSetup;
```

Comment:
```suggestion
  licensing?: LicensingPluginSetup;
```

Since it's declared in optionalPlugins in kibana.jsonc

otherwise we can move it to required plugins.